_guides/dhis2-app-checklist.md

```diff
@@ -0,0 +1,37 @@
+## Starting a new app
+
+### Bootstrap
+
+-   [][create react app](https://github.com/facebook/create-react-app)
+
+## Code style
+
+-   [][dhis2/code-style](https://github.com/dhis2/code-style)
```

Comment:
`dhis2/cli-style`?  Technically there's a github redirect but probably better to use the canonical name...

Reply:
This entire page should be rewritten, it hasn't been published on the site, it has been bouncing around since before d2 and the app-platform.